### DISCUSSION TITLE
UI improvements for small screen/high scaling %

### DISCUSSION THREAD

**Discussion by oorzkws** [ctubio/Krypto-trading-bot#1081] [Ideas]
Currently, the interface is quite cramped if you use large fonts in your browser. This is rectified a bit with the following:
```
.ag-cell {
	text-overflow: clip;
}
```
(removes ellipses from any clipped columns)

However, there's some changes for the table views I'd like thoughts on. Generally, my suggestions are focused on removing information that isn't entirely useful (in my opinion).

Here's a table with the suggested changes:

[X] | _Age | ¯Age | Side | _Price | ¯Price | _Qty | ¯Qty | _Value | ¯Value | Delta
---|-------|------|------|-------|-------|------|------|--------|--------|---
 X  | 5d3h | 5d1h | Ask | 45,681.27 |43,007 | 0.0022 | 0.0022 | 99.81 | 93.97 | 5.84
 X | 1h5m |          | Bid | 40,522.30 |            | 0.582643 | 23,610.03 | 23,502.76 | |

and here's without:
[X] | timePing | timePong | Side | pxPing | pxPong | qtyPing | qtyPong | valPing | valPong | Delta
---|-------|------|------|-------|-------|------|------|--------|--------|---
 X  | 06/03 06:03:57 | 06/03 05:01:26 | Sell | 45681.3 $X | 43007.0 $X | 0.002185 | 0.002185 | 99.8 $X | 94.0 $X | 5.8 $X
 X | 11/03 00:05:03 |          | Bid | 40522.3 $X |  | 0.582643 | 23610.0 $X | 23502.8 $X | |

List of changes:
- Date is now relative, with the previous date format showing on hover
- Fields now use the ping/pong indicators like the orders table
- Fields are re-ordered to put values that would be regularly compared together
- Quote numbers are formatted - full dollar values, no symbols (that information is in the wallet box).
- Base numbers are rounded to the nearest two digits following any zeroes or the decimal point
	- For example: `0.001853426` -> `0.0019`, `5.2436985` -> `5.24`, `6.00725` -> `6.01`
	
Basically what this results in is more easily digestible information, especially for blind folks like me who have their screen scaling at 200%.

Thoughts and suggestions for further improvement?

**Comment by ctubio** [comment]
:koala: neat'¡ you wanna do it or you want me to do it someday?

alsoo.. what is your screen resolution? so we can see the same?

alsoo.. are you sure you scale at 200%? to me 125% is already huge

thank you for your suggestion''' (the UI also hurts my eyes)

**Reply by oorzkws** [reply]
I did give it a quick shot, but my meager JS skills don't seem to translate well to typescript 😥 Looks like the scale is 150% system plus browser zoom, 1080p (attached).

![image](https://user-images.githubusercontent.com/65210810/110241180-5aefc280-7f47-11eb-842c-4ddfbb1f6eb6.png)

**Comment by oorzkws** [comment]
The new changes look good!

**Comment by MiningInMySleep** [comment]
Just thought I'd post to say that the new UI looks really great. Dark mode is super clean and very fresh!
